application/services/telegram.py

```diff
@@ -68,23 +70,22 @@ def request(method, endpoint, params, files):
                 "get" : get
                 }.get(method.lower(), invalid_method)(url, params, files=files)
 
-    if response == "invalid HTTP method" or response.status_code != 200:
-        logging.error(f"Telegram ERROR: {response}")
-        return "error", response
-
-    return "ok", response
+    return response
```

Comment:
Cool! So, I imagine we want to handle this potential 400 response from the caller function? In this case, it would be the error handler :D

Reply:
If so, we should raise an exception when status_code != 200 :D

Reply:
:D

Reply:
Fixed :D